test/testing/fakeauth_test.js

```diff
@@ -32,7 +33,10 @@ suite('fakeauth', function() {
   var server;
 
   suiteSetup(function() {
-    return base.validator().then(function(validator) {
+    return base.validator({
+      folder:  path.join(__dirname, 'schemas'),
+      baseUrl: "http://localhost:1234",
```

Comment:
Can we not just create a validator that contains nothing... that's what the original code did here?